kotlin/reflect/src/main/java/com/squareup/moshi/kotlin/reflect/SpreadWrapper.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.moshi.kotlin.reflect;
+
+import kotlin.reflect.KFunction;
+
+class SpreadWrapper {
```

Comment:
```suggestion
internal class SpreadWrapper {
```

Reply:
`SpreadWrapper` is a `Java` class.  

The reason it is written in `Java` is to avoid using `spread operator`.
I felt that this was a hack that should not be included in this module, so I removed it.